integration_tests/support/commands.ts

```diff
@@ -8,6 +8,7 @@ Cypress.Commands.add(
     initialRequestUrl = paths.HOME,
     hideCookieBanner = true,
   } = {}) => {
+    cy.clearAllCookies()
```

Comment:
Cypress clears browser cookies between tests anyway, but added this to explicitly clear them before logging a user in because logging in / out / in again *within* a test had cookie problems.